core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -552,6 +579,24 @@ static String toString(
         buf.append(", accessLogWriter: ");
         buf.append(accessLogWriter);
         buf.append(')');
+        buf.append(", channelOptions: (");
+        channelOptions.forEach((k, v) -> {
+            buf.append('[');
+            buf.append(k);
+            buf.append(':');
+            buf.append(v);
+            buf.append(']');
+        });
+        buf.append(')');
+        buf.append(", childChannelOptions: (");
+        childChannelOptions.forEach((k, v) -> {
```

Comment:
How about formatting like this:

    childChannelOptions: {a=b, b=c, c=d, ...}